pootle/apps/virtualfolder/models.py

```diff
@@ -194,19 +194,19 @@ def __unicode__(self):
         return self.pootle_path
 
     def save(self, *args, **kwargs):
-        self.pootle_path = self.vfolder.get_adjusted_pootle_path(
-            self.directory.pootle_path
-        )
+        parts = split_pootle_path(self.directory.pootle_path)
+        path_parts = ["", parts[0], parts[1], self.vfolder.name]
+        if parts[2]:
+            path_parts.append(parts[2].strip("/"))
+        path_parts.append(parts[3])
+        self.pootle_path = "/".join(path_parts)
 
         # Trigger the creation of the whole parent tree up to the vfolder
-        # adjusted location.
-        if (self.directory.pootle_path.count('/') >
-                self.vfolder.location.count('/')):
-            parent = VirtualFolderTreeItem.objects.get_or_create(
+        # tp
+        if self.directory.pootle_path.count('/') > 3:
```

Comment:
not sure - but this wont matter once there are no vftis - ill check this bit tho